content/direction/enablement/index.md

```diff
@@ -10,35 +10,39 @@ Our mission is to provide the right technical foundations critical to the succes
 
 ### One year vision
 
-> Remove friction in the install and upgrade process so that customers and prospects have a positive first experience with our product. Our development environment at Sourcegraph enables our team members to solve the most critical problems for our enterprise users. Enable site admins easily configure sourcegraph so that our install base grows.
+> Reduced friction through better processes and tooling, such that both our engineers building Sourcegraph and customer engineers utilizing Sourcegraph are able to provide greater value to our customers, at a greater pace, reliability, and quality.
 
 ### Three year vision
 
-> A frictionless development and deployment environment.
+> A frictionless experience developing and utilizing Sourcegraph, such that we are able to iterate Sourcegraph rapidly, in turn enabling customers to effortlessly experience the value Sourcegraph offers.
```

Comment:
```suggestion
> A frictionless experience developing and utilizing Sourcegraph, such that we can iterate on and release more product value rapidly, in turn enabling customers to experience the value Sourcegraph offers effortlessly.
```

Not 100% sure about the recommended update, but felt like something was missing.